pkg/resources/rhsso.go

```diff
@@ -3,7 +3,7 @@ package resources
 import (
 	"context"
 	"fmt"
-	"github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1"
+	crov1 "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1"
```

Comment:
just out of interest, why the name change here?

Reply:
It's to match the import in the test here: https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/rhsso_test.go#L16